pkg/deploy/elbv2/listener_synthesizer.go

```diff
@@ -3,27 +3,32 @@ package elbv2
 import (
 	"context"
 	awssdk "github.com/aws/aws-sdk-go/aws"
-	elbv2sdk "github.com/aws/aws-sdk-go/service/elbv2"
 	"github.com/go-logr/logr"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/aws/services"
+	"sigs.k8s.io/aws-load-balancer-controller/pkg/deploy/tracking"
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/model/core"
 	elbv2model "sigs.k8s.io/aws-load-balancer-controller/pkg/model/elbv2"
 )
 
-func NewListenerSynthesizer(elbv2Client services.ELBV2, lsManager ListenerManager, logger logr.Logger, stack core.Stack) *listenerSynthesizer {
+func NewListenerSynthesizer(elbv2Client services.ELBV2, trackingProvider tracking.Provider, taggingManager TaggingManager,
+	lsManager ListenerManager, logger logr.Logger, stack core.Stack) *listenerSynthesizer {
 	return &listenerSynthesizer{
-		elbv2Client: elbv2Client,
-		lsManager:   lsManager,
-		logger:      logger,
-		stack:       stack,
+		elbv2Client:      elbv2Client,
+		lsManager:        lsManager,
```

Comment:
seems trackingProvider is not needed here

Reply:
it is not needed, will remove